.lighthouse/jenkins-x/release/recreate-vs.sh

```diff
@@ -12,7 +12,7 @@ declare -a xrepos=(
   # EKS
   "jx3-eks-terraform-vault"
   # local
-  "jx3-kind-vault" "jx3-minikube-vault" "jx3-docker-vault"
+  "jx3-kind-vault" "jx3-minikube-vault" "jx3-docker-vault" "jx3-k3s-vault"
```

Comment:
Are these archived repos? Probably should not be in this list ... will update the PR ...

Reply:
Actually not sure, some repos are archived/removed, some are not, will see what happens to it, and then decide what to do next ...

Reply:
Seems like the other repos are not archived, may be just remove the ones that are